elementary/monitor/api/alerts.py

```diff
@@ -19,27 +19,29 @@ def __init__(self, dbt_runner: DbtRunner, elementary_database_and_schema: str):
         super().__init__(dbt_runner)
         self.elementary_database_and_schema = elementary_database_and_schema
```

Comment:
Let's add `self.config` to the `AlertsAPI`.
This would avoid the pattern of passing the `timezone` around all the functions, and instead just pass it in the `alert_factory_func(timezone=self.config.timezone)` which is more elegant.

Also, this makes more sense as the `query()` function doesn't really have nothing to do with the timezone and it's kind of confusing.

Reply:
added, hope this is the right way
in order to have to confing I'm passing it from the `data_monitoring`